app/utils/Mailer.scala

```diff
@@ -9,16 +9,30 @@ class SES(sesClient: AmazonSimpleEmailService, loginConfig: LoginConfig) {
 
     val uri = loginConfig.tokenReissueUri
 
-    val emailBody = s"<div>Click <a href=$uri$token>here</a> here to obtain a new cookie</div>"
+    val emailBody =
+      s"""
+         |<div>
+         |  Hi,
+         |  <p>
+         |    The Guardian's Editorial Tools are currently experiencing issues with Google Authentication and so we cannot authenticate your access to the Tools.
```

Comment:
Small wording tweak, just to make it more general and hopefully actionable for people:

```
The Guardian's Editorial Tools are experiencing issues and you must perform an additional step to login.

Click here to login and continue using the tools.

Do not share this email with others. The link is private and only for your use.
```

Might sound a bit phishy but I'm thinking we should avoid technical words like `cookie` wherever possible.

Reply:
I think we should continue to mention its a Google issue as it helps to relate the problem with any comms sent from CP.

Reply:
👍

Reply:
Messaging updated if you've got some spare 👀